sdk/nodejs/runtime/rpc.ts

```diff
@@ -640,7 +642,9 @@ export function register<T extends { readonly version?: string }>(source: Map<st
         source.set(key, items);
     }
 
-    log.debug(`registering ${registrationType} ${key}@${item.version}`);
+    if (excessiveDebugOutput) {
+        log.debug(`skip re-registering already registered ${registrationType} ${key}@${item.version}.`);
```

Comment:
Looks like a copy/paste error?

```suggestion
        log.debug(`registering ${registrationType} ${key}@${item.version}`);
```